app/services/allocations/finder.rb

```diff
@@ -73,7 +73,9 @@ def apply_date_range_filters(scope)
     end
 
     def split_params(name)
-      filter_params[name]&.split(',')
+      return if filter_params[name].blank?
+
+      filter_params[name].split(',')
```

Comment:
can't add tests for this as we will never have nil values for locations/status on allocations